docs/source/index.rst

```diff
@@ -40,12 +40,10 @@ The Jupyter notebook
 
 .. toctree::
    :maxdepth: 1
-   :caption: Developer Documentation
+   :caption: Contributor Documentation
 
-   development_intro
```

Comment:
I moved all relevant content from this `development_intro.rst` to `CONTRIBUTING.rst`, making the former obsolete.

If I missed something, please tell me, then I'll add it to the latter.